### PR TITLE
Test zuul

### PR DESCRIPTION
**Pull request by mgirgisf**

### CODE REVIEW

**Comment by mgirgisf** [comment]
recheck

**Comment by raukadah** [comment]
recheck

File: .zuul.yaml [hunk 1]
```diff
@@ -1,6 +1,20 @@
...
     github-check:
       jobs:
-        - noop
+        - feature-verification-tests-osp18
+
+- project:
+    name: infrawatch/feature-verification-tests
+    templates:
+      - feature-verification-tests-osp18-template
+
```
Comment (elfiesmelfie) [inline .zuul.yaml:19]: > Unable to freeze job graph: Job feature-verification-tests-osp18 depends on openstack-k8s-operators-content-provider which was not run.

The content provider is needed, usually it is run with the `podified-multinode-edpm-pipeline` template.
Add openstack-k8s-operators-content-provider to the project or to the template.

Probably better adding it to the template.

File: .zuul.yaml [hunk 2]
```diff
@@ -1,6 +1,23 @@
 ---
-- project:
-    name: infrawatch/feature-verification-tests
+- job:
+    name: feature-verification-tests-osp18
+    parent: telemetry-operator-multinode-autoscaling
+    description: |
```
Comment (elfiesmelfie) [inline .zuul.yaml:5]: ❌ [feature-verification-tests-osp18 ](https://review.rdoproject.org/zuul/build/1fd420ae5ac243a9b310924f6ea6b118)ERROR Project github.com/openstack-k8s-operators/dataplane-operator does not have the default branch master in 5s

You will need to add the dataplane operator to the job definition with the override checkout

Either you can add it here, or in the telemetry-job-multinode-autoscaling definition in telemetry-operator.

required-projects:
    - name: openstack-k8s-operators/dataplane-operator
       override-checkout: main 
       
Example is here: https://github.com/infrawatch/service-telemetry-operator/blob/853e4574b5d3c93faa2b9001f8bf18f0a9eddc9e/.zuul.yaml#L31C1-L39C32

**Pull request closed without merging**